Examples/drawing-benchmark.lisp

```diff
@@ -19,104 +23,276 @@
+      (make-pane 'toggle-button :label "indirect" :id :indirect)
+      (make-pane 'toggle-button :label "pattern" :id :pattern)
+      (make-pane 'toggle-button :label "pattern*" :id :pattern*)))
+   (grid-size
+    (with-radio-box ()
+      (radio-box-current-selection
+       (make-pane 'toggle-button :label "1" :id 1))
+      (make-pane 'toggle-button :label "2" :id 2)
+      (make-pane 'toggle-button :label "3" :id 3)
+      (make-pane 'toggle-button :label "4" :id 4)
+      (make-pane 'toggle-button :label "5" :id 5)))
+   (recording
+    (with-radio-box ()
+      (radio-box-current-selection
+       (make-pane 'toggle-button :label "Off" :id 'off))
+      (make-pane 'toggle-button :label "On" :id 'on)))
+   (transform-ops
     (with-radio-box ()
       (radio-box-current-selection
-       (make-pane 'toggle-button :label "random" :id :random))
-      (make-pane 'toggle-button :label "red" :id +red+)
-      (make-pane 'toggle-button :label "flipping ink" :id +flipping-ink+)
-      (make-pane 'toggle-button :label "indirect" :id +foreground-ink+)
-      (make-pane 'toggle-button :label "pattern" :id *glider*)
-      (make-pane 'toggle-button :label "pattern*" :id *transformed-glider*))))
+       (make-pane 'toggle-button :label "Scale do/undo" :id 'scale-ident))
+      (make-pane 'toggle-button :label "Scale accumulative" :id 'scale-accum)
+      (make-pane 'toggle-button :label "Xlate do/undo" :id 'xlate-ident)
+      (make-pane 'toggle-button :label "Xlate accumulative" :id 'xlate-accum)
+      (make-pane 'toggle-button :label "Scale & Xlate do/undo" :id 'sx-ident)
+      (make-pane 'toggle-button :label "Scale & Xlate accum." :id 'sx-accum)))
+   (max-transforms
+    (with-radio-box ()
+      (radio-box-current-selection
+       (make-pane 'toggle-button :label "0" :id 0))
+      (make-pane 'toggle-button :label "4" :id 4)
+      (make-pane 'toggle-button :label "16" :id 16)
+      (make-pane 'toggle-button :label "64" :id 64)))
+   (results :application
+            :min-width 600
+            :min-height 75
+            :end-of-line-action :scroll
+            :incremental-redisplay nil
+            :display-time t))
   (:layouts
    (default
     (vertically ()
       (horizontally ()
-        (labelling (:label "Mode") mode)
-        (labelling (:label "Ink") ink))
-      canvas))))
-
-(defmethod run-drawing-benchmark (frame stream)
-  (window-clear stream)
-  (let* ((width (rectangle-width (sheet-region stream)))
-         (height (rectangle-height (sheet-region stream)))
-         (mode (gadget-id (gadget-value (find-pane-named frame 'mode))))
-         (ink (gadget-id (gadget-value (find-pane-named frame 'ink))))
+        (vertically ()
+          (horizontally ()
+            (labelling (:label "Shape") shape)
+            (labelling (:label "Grid Size") grid-size)
+            (labelling (:label "Ink") inking)
+            (labelling (:label "Output Recording") recording))
+          (labelling (:label "Co-ord Transformations")
```

Comment:
please use full names, I think that in this particular case
"Transformations" should be just fine.